vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.java

```diff
@@ -1358,8 +1419,10 @@ private void refreshPushersList() {
      * Refresh the emails list
      */
     private void refreshEmailsList() {
+        final List<ThirdPartyIdentifier> currentEmail3PID = mSession.getMyUser().getlinkedEmails();
+
```

Comment:
You should create a copy of the list (it can be either in the SDK or here).
It avoids crash when the list is updated while enumerating it).